omigami/feature_selector.py

```diff
@@ -188,7 +191,10 @@ def fit(
             b.update(progress)
             for _ in range(self.n_repetitions):
                 data_splitter = DataSplitter(
-                    self.n_outer, self.n_inner, input_data, self.random_state,
+                    self.n_outer,
```

Comment:
did you black these?

Reply:
yes I ran black on this file

Reply:
I added black as requirements following Alan's new confluence section on this

Reply:
Yes it's good, I'm just surprised cause I black automatically at saving and it was not breaking those on multiple lines. I might have to review my environment configuration